lang/java/compiler/src/main/java/org/apache/avro/compiler/schema/SchemaVisitor.java

```diff
@@ -25,31 +25,31 @@
    * Invoked for schemas that do not have "child" schemas (like string, int ...)
    * or for a previously encountered schema with children, which will be treated
    * as a terminal. (to avoid circular recursion)
-   * 
+   *
    * @param terminal
-   * @return
+   * @return SchemaVisitorAction
```

Comment:
I'd get rid of these `@return` tags (which also gets rid of the warning), instead of returning "non-documentation"!